contracts/GSN/IRelayHub.sol

```diff
@@ -50,38 +80,58 @@ contract IRelayHub {
         Removed    // The relay has been removed by its owner and can no longer relay calls. It must wait for its unstakeDelay to elapse before it can unstake
     }
 
-    // Returns a relay's status. Note that relays can be deleted when unstaked or penalized.
+    /**
+     * @dev Returns a relay's status. Note that relays can be deleted when unstaked or penalized, causing this function
+     * to return an empty entry.
+     */
     function getRelay(address relay) external view returns (uint256 totalStake, uint256 unstakeDelay, uint256 unstakeTime, address payable owner, RelayState state);
 
     // Balance management
 
-    // Deposits ether for a contract, so that it can receive (and pay for) relayed transactions. Unused balance can only
-    // be withdrawn by the contract itself, by callingn withdraw.
-    // Emits a Deposited event.
+    /**
+     * @dev Deposits ether for a contract, so that it can receive (and pay for) relayed transactions.
```

Comment:
Assume Ether should be capitalized
```suggestion
     * @dev Deposits Ether for a contract, so that it can receive (and pay for) relayed transactions.
```